tests/src/constructor-validation/version-4.ts

```diff
@@ -0,0 +1,7 @@
+import { NearBindgen, LookupMap } from "near-sdk-js";
```

Comment:
nit: let's name these files according to what they are. Like: `NoConstructorContract.ts`